lib/modules/datasource/nuget/__snapshots__/index.spec.ts.snap

```diff
@@ -1497,6 +1497,7 @@ exports[`modules/datasource/nuget/index getReleases processes real data (v3) nus
       "version": "3.12.0",
     },
   ],
+  "sourceUrl": "https://nunit.org/",
```

Comment:
This is better suited for homepage  and not sourceUrl

Reply:
It's the same behavior as in 
`modules/datasource/nuget/index getReleases processes real data (v2) 1`
`modules/datasource/nuget/index getReleases processes real data (v3) feed is not a nuget.org 1`
`modules/datasource/nuget/index getReleases processes real data with no github project url (v2) 1`